src/resources/js/tribe-events.js

```diff
@@ -776,10 +776,7 @@ Date.prototype.format = function( mask, utc ) {
 				! tribe_ev.state.view ||
 
 				// We are on the default mobile view
-				tribe_ev.data.default_mobile_view == tribe_ev.state.view ||
-
-				// We are with an defined view
-				tribe_ev.data.cur_url == tribe_ev.data.base_url
+				tribe_ev.data.default_mobile_view == tribe_ev.state.view
```

Comment:
Did you run `gulp` command on this PR? To generate the minified JS file?

Reply:
I did not, thanks for flagging this will send in a moment.

Reply:
File has been added ;)